bugbug/bugzilla.py

```diff
@@ -29,6 +30,8 @@
 
 COMMENT_INCLUDE_FIELDS = ["id", "count", "text", "author", "creation_time"]
 
+PRODUCT_COMPONENT_CSV_REPORT = "https://bugzilla.mozilla.org/report.cgi?f1=creation_ts&o1=greaterthan&resolution=---&x_axis_field=product&y_axis_field=component&width=600&height=350&action=wrap&ctype=csv&format=table"
```

Comment:
From the query, we can remove:
- resolution=---, as we also want to consider resolved bugs
- width=600&height=350, I guess we don't need them for the csv format

We should add a date limit to reduce the cost, maybe past 6 months. You just need to add a v1, as you already have f1 and o1. v1 can be in the format like 'YYYY-MM-DD'.

Reply:
Got it, this means we will likely have to dynamically generate the URL